tests/unit/test_allowlist.py

```diff
@@ -46,39 +46,151 @@ def allowlist():
     return w
 
 
+@pytest.fixture(scope="module")
+def allowlist_entries():
+    entries = {
+        "github.com": flexmock(
+            id=1, namespace="github.com", status=AllowlistStatus.denied.value
+        ),
+        "gitlab.com": flexmock(
+            id=2, namespace="gitlab.com", status=AllowlistStatus.denied.value
+        ),
+        "github.com/fero": flexmock(
+            id=3,
+            namespace="github.com/fero",
+            status=AllowlistStatus.approved_manually.value,
+        ),
+        "gitlab.com/lojzo": flexmock(
+            id=4,
+            namespace="gitlab.com/lojzo",
+            status=AllowlistStatus.approved_automatically.value,
+        ),
+        "github.com/konipas": flexmock(
+            id=5,
+            namespace="github.com/konipas",
+            status=AllowlistStatus.waiting.value,
+        ),
+        "gitlab.com/packit-service": flexmock(
+            id=6,
+            namespace="gitlab.com/packit-service",
+            status=AllowlistStatus.denied.value,
+        ),
+        "gitlab.com/packit": flexmock(
+            id=8,
+            namespace="gitlab.com/packit",
+            status=AllowlistStatus.approved_manually.value,
+        ),
+        "github.com/packit": flexmock(
+            id=10,
+            namespace="github.com/packit",
+            status=AllowlistStatus.denied.value,
+        ),
+        "gitlab.com/packit-service/src": flexmock(
```

Comment:
could we get another layer for gitlab here? e.g. `gitlab.com/packit-service/src/systemd`

Reply:
...so what should happen here actually for gitlab.com?

Reply:
> could we get another layer for gitlab here? e.g. `gitlab.com/packit-service/src/systemd`

test? sure

> ...so what should happen here actually for gitlab.com?

nothing, it just looks it up as parent namespace that can be approved; this could be potentially helpful with self-hosted instances with restricted access (e.g. for testing), it would allow allowlisting whole instance

Reply:
yup, as a test case

thanks for explanation